tests/expressions/tests.py

```diff
@@ -387,6 +387,116 @@ def test_ticket_18375_chained_filters(self):
         )
         self.assertEqual(str(qs.query).count('JOIN'), 2)
 
+    def test_outer_ref(self):
+        inner = Company.objects.filter(point_of_contact=OuterRef('pk'))
+        outer = Employee.objects.annotate(is_point_of_contact=Exists(inner))
+
+        error_message = 'This queryset contains a reference to an outer query, and may only be used in a subquery.'
+        with self.assertRaisesMessage(ValueError, error_message):
+            inner.exists()
+
+        self.assertTrue(outer.exists())
+
+    def test_subquery(self):
+        Company.objects.filter(name="Example Inc.").update(
+            point_of_contact=Employee.objects.get(firstname="Joe", lastname="Smith"),
+            ceo=Employee.objects.get(firstname="Max", lastname="Mustermann"),
+        )
+        Employee.objects.create(firstname="Bob", lastname="Brown", salary=40)
+        self.maxDiff = 1200
```

Comment:
Hahaha. Guess that was left from when I was still debugging something. I didn't even see that line in there!